src/HornedBeast.js

```diff
@@ -1,13 +1,33 @@
 import {Component} from 'react';
+import heart from './images/heart.png';
 
 class HornedBeast extends Component {
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      timesClicked: 0,
+    };
+  }
+
+  handleClick = () => {
+    this.setState({timesClicked: this.state.timesClicked + 1});
+  }
+
   render() {
-    console.log('imageUrl in HB: ', this.props.imageUrl);
     return(
       <>
-        <h2>{this.props.title}</h2>
-        <img src={this.props.imageUrl} alt={this.props.description} title={this.props.title}/>
-        <p>{this.props.description}</p>
+        <div class="beast-wrapper">
```

Comment:
See line 20: `className` should be used instead of `class` :) This happens sometimes, I know my brain for sure does this sometimes 👍